src/main/java/phrasecount/pojos/PhraseAndCounts.java

```diff
@@ -0,0 +1,24 @@
+package phrasecount.pojos;
+
+public class PhraseAndCounts extends Counts {
+  public String phrase;
+
+  public PhraseAndCounts(String phrase, int docCount, int sum) {
```

Comment:
`sum` is used here for the total count of phrases across all documents but it is named `total` in `Counts` object.  How about using the same name?  Also, `phraseCount` might be more descriptive than sum or total.